docs/reference/changelog-r2021.md

```diff
@@ -26,6 +26,7 @@ Released on June, Xth, 2021.
       - Exposed the `robot_description` ROS parameter (activated through the `--robot-description` flag) that contains the URDF of the robot.
   - New Samples:
     - Added a simple room with a Nao robot ([#2701](https://github.com/cyberbotics/webots/pull/2701)).
+    - Added HingeJointWithBacklash proto that extends [HingeJoint](hingejoint.md) to model the effect of backlash and a corresponding sample world ([#2786](https://github.com/cyberbotics/webots/pull/2786)).
```

Comment:
We should also add a line about the new fields of the HingeJointParameters (stopEPF and stopCFM).

Reply:
Right, I noticed I didn't update the `HingeJointParameters` docs about that addition either so I fixed it now.